content/en/docs/setup/keadm.md

```diff
@@ -42,6 +44,36 @@ Kubernetes version verification passed, KubeEdge installation will start...
 KubeEdge cloudcore is running, For logs visit:  /var/log/kubeedge/cloudcore.log
 ```
 
+### keadm beta init
+
+We now provide a solution for integrating Cloudcore Helm Chart with `keadm beta init`. 
+
+Example:
+
+```shell
+# keadm beta init --advertise-address="THE-EXPOSED-IP" [--set cloudcore-tag=v1.9.0] --profile version=v1.9.0 --kube-config=/root/.kube/config
```

Comment:
What's meaning of `[]` for `--set`? And if we want to specify the kubeedge version, we need to modify both the `[--set cloudcore-tag=v1.9.0]` and `--profile version=v1.9.0`?

Reply:
Set flags would override the value from values.yaml with profile. I would remove these confusing values.